tmt/steps/discover/fmf.py

```diff
@@ -141,7 +192,12 @@ class DiscoverFmfStepData(tmt.steps.discover.DiscoverStepData):
         default=False,
         option='--dist-git-merge',
         is_flag=True,
-        help='Merge copied sources and plan fmf root.',
+        help="""
+            Set to ``true`` to combine fmf root from the sources and fmf root from the plan.
+            It allows to have plans and tests defined in the DistGit repo which use tests
+            and other resources from the downloaded sources. Any plans in extracted sources
+            will not be processed.
+            """,
     )
```

Comment:
I've noticed there are a couple of undocumented fields:

```
repository:
Default: not set

revision:
Default: not set

upgrade-path:
Default: not set
```

Shall we include at least a short obsoletion / explanation note for them? Probably in a separate pull request?

Reply:
@psss so those fields are obsolete? Mmm yeah I think any update on those can be done in a different PR